platform/ios/benchmark/MBXBenchViewController.mm

```diff
@@ -1,6 +1,6 @@
 #import "MBXBenchViewController.h"
 
-#import <mbgl/ios/MGLMapView.h>
+#import "MGLMapView.h"
```

Comment:
If this depends on the iossdk target, which is a dynamic framework, you should be able to import the umbrella header, as `<Mapbox/Mapbox.h>`.